### ISSUE TITLE
Update image to use gcr.io/distroless/static:nonroot as base for final stage

### ISSUE THREAD

**Issue by krancour** [brigadecore/brigade-cron-event-source#43]

**Comment by hamees-sayed** [comment]
Am I supposed to change this:
`image:
  repository: brigadecore/brigade-cron-event-source`
  
  to this:
  `image:
    repository: gcr.io/distroless/static:nonroot`
    
 source: `[brigade-cron-event-source](https://github.com/brigadecore/brigade-cron-event-source/tree/1fac8fa8973dc16e8a3787ff60977876c61bbfef)/[charts](https://github.com/brigadecore/brigade-cron-event-source/tree/1fac8fa8973dc16e8a3787ff60977876c61bbfef/charts)/[brigade-cron-event-source](https://github.com/brigadecore/brigade-cron-event-source/tree/1fac8fa8973dc16e8a3787ff60977876c61bbfef/charts/brigade-cron-event-source)/values.yaml`

**Comment by krancour** [comment]
No... it would be a change to the second (final) `FROM` directive in the `Dockerfile`.

**Comment by hamees-sayed** [comment]
Like this: 

`FROM` gcr.io/distroless/static:nonroot

**Comment by hamees-sayed** [comment]
I got it
